settings.py

```diff
@@ -93,3 +93,4 @@
 
 DEFAULT_FROM_EMAIL = 'no-reply@example.com'
 CONTACT_EMAIL = 'info@edx.org'
+TECH_SUPPORT_EMAIL = 'technical@example.com'
```

Comment:
@catong Good question. This is just a test settings file needed for this repo. In production, this will be configured to go to the correct tech support address. 

Is that your question, or are you suggesting that there is a difference between tech support and learner support?